install-and-configure/install/multi-cluster/federated-etl/aggregator.md

```diff
@@ -59,19 +63,18 @@ serviceAccount:
 
 ### Aggregator Optimizations
 
-For larger deployments of Kubecost, Aggregator can be tuned.
+For larger deployments of Kubecost, Aggregator can be tuned. The settings below are in addition to the basic configuration above.
+
+{% hint style="info" %}
+This configuration is estimated to be sufficient for environments monitoring < 60k unique containers per day. You can check this metric on the `/diagnostics` page.
+{% endhint %}
```

Comment:
@bstuder99 @ameijer Added this line to give users an estimate for what scale these configurations should be sufficient for. We'll need to continuously update this as we learn more from users. I added a similar line to the "Basic configuration" section.

If we're good with this, we can approve & merge the PR!